includes/gateways/class-wc-subscriptions-gateway-restrictions-manager.php

```diff
@@ -46,6 +46,24 @@ public static function enqueue_scripts() {
 				'decimal_point_separator'      => $decimal_separator,
 			);
 
+			/**
+			 * Allow WC Payments to return the minimum amount that can be processed.
+			 *
+			 * @since 1.5.0
+			 *
+			 * @param bool|float The minimum amount that can be processed in the given currency.
+			 * @param string.    The currency.
+			 */
+			$minimum_processable_amount = apply_filters( 'woocommerce_subscriptions_minimum_processable_recurring_amount', false, get_woocommerce_currency() );
+
+			if ( $minimum_processable_amount ) {
+				$i18n_minimum_price                         = sprintf( get_woocommerce_price_format(), get_woocommerce_currency_symbol(), number_format( $minimum_processable_amount, $decimals, $decimal_separator, '' ) );
+				$script_data['minimum_subscription_amount'] = $minimum_processable_amount;
+
+				// Translators: Placeholder is a localized price string (eg. $1.00).
+				$script_data['i18n_below_minimum_subscription_error'] = sprintf( __( "Warning! Your store's payment method cannot process a recurring total below %s", 'woocommerce-subscriptions' ), $i18n_minimum_price );
```

Comment:
The wording on this feels a little strange to me but I can't think of a better alternative. I want to write a whole paragraph 😅 

What are your thoughts on removing "payment method" to shorten the message but also remove some questions it might create like "which one of my payment methods"? I also provided an alternative than using "recurring total" but I'm not sure about it haha

Since this is just wording, I'm happy to go with whatever

```suggestion
				$script_data['i18n_below_minimum_subscription_error'] = sprintf( __( "Warning! Your store cannot create subscriptions less than %s.", 'woocommerce-subscriptions' ), $i18n_minimum_price );
```